### DISCUSSION TITLE
Computed information from previous notice information

### DISCUSSION THREAD

**Discussion by guascce** [eprocurementontology/eprocurementontology#491] [Q&A]
Several elements can be derived from the linked chain of previous notice.
**PurchaseContracts**
In order to compute the indicators, it is needed to know which Contract Award Notice is for a PurchaseContract. Will the conversion instantiate epo:PurchaseContracts in such a case for AWARD_CONTRACT/CONTRACT_NO?
**Termination/cancellation**
When a notice is cancelled how is the termination notice defined in ePO? 
For now, ePO does not manage business object statuses. Specifically procedure does not have a status neither Notice . This can create a problem for users who will need very complex queries to discover not cancelled procedures.

**Comment by muricna** [comment]
@ignaciobavastro


We currently map the CAN  in the actual CAN is not possible to identify whether the contract is a purchase contract or not.

With regard to the cancellation of a notice, I am not aware of such a possiblity could you please provide an example.
